spec/acceptance/content_standalone_mirror_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper_acceptance'
+
+describe 'pulpcore mirror' do
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<~PUPPET
+      include certs::foreman_proxy
+
+      class { 'foreman_proxy':
+        register_in_foreman => false,
+        ssl_ca              => $certs::foreman_proxy::proxy_ca_cert,
+        ssl_cert            => $certs::foreman_proxy::proxy_cert,
+        ssl_key             => $certs::foreman_proxy::proxy_key,
+      }
+
+      class { 'foreman_proxy_content':
+        pulpcore_mirror => true,
+        # No Ansible repo is set up in our CI
+        enable_ansible  => false,
+      }
+      PUPPET
+    end
+  end
+
+  describe service('httpd') do
+    it { is_expected.to be_running }
+    it { is_expected.to be_enabled }
+  end
+
+  describe file('/etc/httpd/conf.d/10-pulpcore.conf') do
+    it { is_expected.to be_file }
+    it { is_expected.to contain(%r{DocumentRoot "/var/lib/pulp/pulpcore_static}) }
+  end
+
+  describe file('/etc/httpd/conf.d/10-rhsm-pulpcore-https-443.conf') do
+    it { is_expected.to be_file }
+    # TODO: this is a regression introduced in 76e2a6852d1d2ca33935ccf8a6ab69992c32ec1d
+    it { is_expected.to contain(%{DocumentRoot "/var/www}) }
```

Comment:
Prior to this we had the Pulp static docroot, just like the HTTP config. I was looking at it and saw we always had `/var/www` as the docroot in the reverse proxy config, but then we proxied `/`. However, with Pulp this is not the case and we may expose files. `/var/www` is not intended to be a docroot. Debian does have it as a default, but on Red Hat it's supposed to be `/var/www/html`. This could leak files. However, the `pulpcore_static` is supposed to be the docroot, see https://github.com/theforeman/puppet-pulpcore#installation-layout= as well.

I hadn't gotten around to writing up a proper issue but I will.

Reply:
Okay. I think this shouldn't block this PR.

May I ask you to update this comment with a reference, once you have it?